static/app/views/preprod/sidebar/buildDetailsSidebarAppInfo.tsx

```diff
@@ -61,6 +68,14 @@ export function BuildDetailsSidebarAppInfo(props: BuildDetailsSidebarAppInfoProp
           </InfoIcon>
           <InfoValue>{getReadableArtifactTypeLabel(appInfo.artifact_type)}</InfoValue>
         </InfoRow>
+        {appInfo.is_installable && (
```

Comment:
IMO I would always show this but have the text update to "Installable" or "Not installable". Otherwise if someone unfamiliar with build distribution views the page it might not be obvious to them.